build-tools/automation/yaml-templates/plots-to-appinsights.yaml

```diff
@@ -0,0 +1,63 @@
+parameters:
+  framework: net472     # TargetFramework setting for the build-tools/plots-to-appinsights/ProcessPlotCSVFile source project that becomes part of the build output directory
```

Comment:
Optionally, you can add `<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>` to the `csproj` since we aren't multi-targeting this project and then it won't be nested in a `net472` output directory and this can be killed.

Reply:
Excellent suggestion and will avoid maintenance headaches down the road.  Addressed with commit: f4a541069b53112d50dbae1395449f840eb14dd0